### DISCUSSION TITLE
SaveFileDialog.ShowAsync crashes on trimmed .net6.0 rc1 build

### DISCUSSION THREAD

**Discussion by ahopper** [AvaloniaUI/Avalonia#6609] [General]
SaveFileDialog.ShowAsync crashes on trimmed .net6.0 rc1 or p7 build on windows

adding `<BuiltInComInteropSupport>true</BuiltInComInteropSupport>` to csproj fixes it.

I wonder if this should be added to the templates.

**Comment by kekekeks** [comment]
We should probably migrate dialogs to our manual COM interop layer, especially since it now works with NativeAOT
